products/opensuse/main.pm

```diff
@@ -184,7 +184,11 @@ sub load_boot_tests {
         loadtest "installation/bootloader_uefi";
     }
     elsif (get_var("IPMI_HOSTNAME")) {    # abuse of variable for now
-        loadtest "installation/qa_net";
+#        loadtest "installation/qa_net";
+# replace IPMI as PXE Boot moth
+#        set_var("DELAYED_START", "1");
+#        loadtest "autoyast/pxe_boot";
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit

Reply:
Got it, thank you for your great suggestion and comments.